terraform/modules/shared/main.tf

```diff
@@ -472,8 +472,49 @@ resource "azurerm_kubernetes_cluster" "cluster" {
   }
 
   identity {
-    type         = "UserAssigned"
-    identity_ids = [azurerm_user_assigned_identity.pipeline_runner.id]
+    type = "SystemAssigned"
+  }
```

Comment:
SystemAssigned for now is probably fine (and probably the right place to start anyway), but if we end up needing to configure additional roles for the cluster identity it may end up being easier if it UserAssigned.

Reply:
That makes sense, will keep it in mind going forward. We did try it with the UserAssigned pipeline identity, but it did mean Helm wasn't able to authenticate to the K8s cluster - I'm sure there's more debugging we could do on that front though.